src/content/en/tools/workbox/modules/workbox-google-analytics.md

```diff
@@ -164,3 +164,9 @@ first custom metric, the parameter name would be "cm1":
       },
     });
     </pre>
+
+## Testing Workbox Google Analytics
+
+As Workbox Google Analytics uses Background Sync to replay events, it can
+be unintuitive to test. Read more at [Testing Workbox Background Sync]
```

Comment:
Sadly our MD parser gets cranky when you split links over multiple lines. I'll fix, but in future, keep links on a single line.